### ISSUE TITLE
EC: Use loss function value for early stopping rather than target metrics?

### ISSUE THREAD

**Issue by klieret** [gnn-tracking/hyperparameter_optimization#44]

**Comment by klieret** [comment]
Would only work if we leave focal loss HPs unchanged
